app/controllers/application_controller/ci_processing.rb

```diff
@@ -220,9 +220,7 @@ def check_retire_requirements(selected_items)
     return true if %w(orchestration_stack service).include?(controller_name)
 
     if VmOrTemplate.find(selected_items).any? { |vm| !vm.supports_retire? }
-      add_flash(_("Retire does not apply to selected %{model}") %
-        {:model => ui_lookup(:table => "miq_template")}, :error)
-      javascript_flash(:scroll_top => true)
+      javascript_flash(:text => _("Retire does not apply to selected item"), :severity => :error, :scroll_top => true)
```

Comment:
@martinpovolny while testing, found another place that shows similar flash message when selecting a VM from list view and pressing "Set Retirement Dates" button https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/mixins/actions/vm_actions/retire.rb#L25